### DISCUSSION TITLE
My submission deadline was only 36 minutes after I got the GitHub classroom invitation.

### DISCUSSION THREAD

**Discussion by yopknopixx** [community/community#64194] [GitHub Education]
### Hi everyone,

I recently applied to Ivy Octernship via GitHub and I got an invitation to collaborate on a private repo via email. However, when I checked the assignment due date it was only 36 minutes after I got the mail. Is this expected behavior? If not how can this be corrected?

**Comment by Khushiyant** [comment]
Actually, every project has mentioned their deadline so If you apply on last minute, your will have `total time = final_deadline - today_date`

**Reply by yopknopixx** [reply]
The deadline was moved from the 16th to the 23rd of August by the project. The deadline displayed on the dashboard is correct however, when I click `Review Assignment Due Date` on the repo README, it still shows the 16th of August.
